machine_learning_hep/utilities.py

```diff
@@ -116,34 +116,26 @@ def create_folder_struc(maindir, listpath):
         if not os.path.exists(folder):
             os.makedirs(folder)
 
-def checkdirlist(dirlist):
-    """
-    Checks if list of folder already exist, to not overwrite by accident
-    """
-    exfolders = 0
-    for _, mydir in enumerate(dirlist):
-        if os.path.exists(mydir):
-            print("rm -rf ", mydir)
-            exfolders = exfolders - 1
-    return exfolders
-
-def checkdir(mydir):
+def checkdir(mydir, V=True):
     """
     Checks if folder already exist, to not overwrite by accident
     """
     exfolders = 0
     if os.path.exists(mydir):
+        if V:
+          print("Please remove this directory:")
```

Comment:
I am not sure this is very useful. In the current version, one can copy the block of lines from the standard output and paste it directly in the command line to delete the directories. This change would add this comment between each pair of lines which would make it less smooth to execute the rm commands for all the directories.

Reply:
Also I think one letter variable names are not allowed in the style check because they are not very meaningful most of the time.

Reply:
Also here, I agree, it was just to know what the code was telling